src/rkc_utils.jl

```diff
@@ -1,10 +1,10 @@
 # This function calculates the largest eigenvalue
 # (absolute value wise) by power iteration.
-const RKCAlgs = Union{RKC,IRKC}
+const RKCAlgs = Union{RKC,IRKC,ESERK5}
```

Comment:
isn't this already used to dispatch to predictive?

Reply:
No. I think you might be confusing it with this line `const PredictiveControllerAlgs = Union{RKC,RadauIIA5}`